spec/models/patient_summary_spec.rb

```diff
@@ -58,6 +58,10 @@
         expect(patient_summary.district).to eq(patient.address.district)
         expect(patient_summary.state).to eq(patient.address.state)
       end
+
+      it "includes patient status" do
+        expect(patient_summary.status).to eq(patient.status)
+      end
```

Comment:
Non-blocking comment: Hmm, what do you think of lumping this together with name and gender, as `it "includes patient attributes"`?